src/createServer.js

```diff
@@ -1,3 +1,63 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normUrl = new URL(req.url, 'http://localhost:5700');
+    const text = normUrl.pathname.slice(1);
+    const toCase = normUrl.searchParams.get('toCase');
+    const errorsArray = [];
+
+    try {
+      if (!text) {
+        errorsArray.push({
+          message:
+            'Text to convert is required. Correct request is: ' +
+            '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+        });
+      }
+
+      if (!toCase) {
+        errorsArray.push({
+          message:
+            '"toCase" query param is required. Correct request is: ' +
+            '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+        });
+      }
+
+      if (
+        !['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(
+          toCase.toUpperCase(),
```

Comment:
- Create a function that will validate data and move it to a separate file
- Create a constant for available cases - `['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER']`, and also use it to generate an error message regarding available cases